__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const e = require("express");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("200: returns an array objects on the key of topics with following properties: 'slug' and 'description'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics.length).toBe(3);
+        body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("400: returns 'not found' when api topics endpoint is invalid due to typo or different endpoint received", () => {
+    return request(app)
+      .get("/api/tpics")
+      .expect(404)
+      .then(({body}) => expect(body.msg)
+      .toBe("Not Found"));
+  });
```

Comment:
Really nice test, but the test description is not correct, this should actually be a 404 which you have asserted for correctly but just dont have the correct description for it. This test should also not be in the describe block for /api/topics as this is just a generic invalid endpoint and not related to /api/topics at all.